packages/grafana-data/src/types/fieldOverrides.ts

```diff
@@ -88,7 +88,7 @@ export interface ApplyFieldOverrideOptions {
   custom?: FieldConfigEditorRegistry;
 }
 
-export enum StandardFieldConfigProperties {
+export enum StandardFieldConfigProperty {
```

Comment:
Alternative name ideas
- StandardFieldConfigId  (kind of like this one) 
- StandardFieldConfigPropertyName (too long) 
- FieldConfigPropertyName

Reply:
Maybe `FieldConfigPropertyId`?  it is standard by default?

Reply:
Or just `FieldConfigProperty`?  we only use "property" to refer to the keys on field config so maybe we don't need id?

Reply:
I think `FieldConfigProperty` would work great!